MapboxVision/src/main/java/com/mapbox/vision/VisionManager.kt

```diff
@@ -69,6 +77,11 @@ object VisionManager : BaseVisionManager {
 
     private var isRecording = false
 
+    private val handlerMain = Handler(Looper.getMainLooper())
+
+    // TODO temporary fix, this should be moved to core
```

Comment:
Mind filing an issue for that in guthub?

Reply:
yep